fix(ET-1265): Updates migration service documentation (#5908)
EnterpriseDB/docs: product_docs/docs/transporter/ba_preview/index.mdx

indexCards: simple
deepToC: true
directoryDefaults:

>   description: "EDB Transporter is our PG AI integrated migration solution based on Debezium and Kafka. Change-event data streaming enables secure, fault-tolerant, and performant migrations to BigAnimal, on-premise EPAS/Postgres, or other cloud-hosted databases."

Reviewer: No use of first person as per style guide.

Also this is somewhat long for a description which is meant to be in an index card box.

Also, ~BigAnimal~ - EDB Postgres AI Cloud Service (or Cloud Service)
```suggestion
  description: "EDB Transporter is a PG AI integrated migration solution enabling secure, fault-tolerant, and performant migrations to EDB Postgres AI Cloud Service, on-premise EPAS/Postgres, or other cloud-hosted databases."
```
